tools/ingest2parquet/Makefile

```diff
@@ -3,22 +3,27 @@ REPOROOT=../../
 
 include $(REPOROOT)/.make.defaults
 
-DOCKER_IMAGE_NAME=ingest2parquet
-DOCKER_IMAGE_VERSION=0.1
-DOCKER_IMAGE=${DOCKER_REGISTRY_ENDPOINT}/$(DOCKER_IMAGE_NAME):$(DOCKER_IMAGE_VERSION)
+include ../../transforms/.make.transforms
+include ../../.make.versions
 
+TRANSFORM_NAME=ingest2parquet
+DOCKER_IMAGE_VERSION=${INGEST_TO_PARQUET_VERSION}
 
-venv:	.defaults.lib-src-venv
 
-build:	 venv image
+venv::	.transforms.venv
```

Comment:
This makefile should no longer reference .transforms - I believe I've moved common targets to .make.defaults. . For example, this should be .defaults.lib-src-venv instead of .transforms.venv.

Reply:
Probably
`
venv:: .defaults.lib-src-venv `

Reply:
fixed